### PR TITLE
docs: add example YAML for workflow_dispatch

### PR DESCRIPTION
**Pull request by bnb**
### Why: 

This page explains workflow_dispatch - it shows how to actually use the UI, but it's easy to land here wanting to know how to fully use it both in terms of UI and code. This adds a simple example of the syntax for how to use it.

### What's being changed:

Adding a H3 (aligned with the other headings) and a small code example.

### Check off the following:
- [x] All of the tests are passing.
- [x] I have reviewed my changes in staging.
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Comment by janiceilene** [comment]
Thanks for opening a PR 👍 Please fill out the pull request template and we'll get this up for review!

**Comment by bnb** [comment]
Did my best with the review of localization and content style, apologies if I missed anything - happy to make whatever changes necessary if I missed something 🙇🏻

**Comment by janiceilene** [comment]
Thanks @bnb Can you fill out the `Why` and the `What's being changed?` in your original comment in this PR? That'll help us to triage and find the best reviewers for your pull request. Thank you so much!

**Comment by bnb** [comment]
My apologies @janiceilene, should be fixed now, I totally spaced not once but twice. Thank you for being patient with me 🙇🏻

**Comment by janiceilene** [comment]
Thank you so much @bnb! I think the format of the PR template wasn't the best. The big warning at the top made it super easy to bypass completely. I tweaked it this morning, so hopefully it's a bit more welcoming! I'll get this triaged for review 💝

File: content/actions/managing-workflow-runs/manually-running-a-workflow.md [hunk 1]
```diff
@@ -32,3 +32,11 @@ To trigger the `workflow_dispatch` event on {% data variables.product.prodname_d
 When using the REST API, you configure the `inputs` and `ref` as request body parameters. If the inputs are omitted, the default values defined in the workflow file are used. 
 
 For more information about using the REST API, see the "[Create a workflow dispatch event](/rest/reference/actions/#create-a-workflow-dispatch-event)."
+
+### Example YAML
```
Comment (rachmari) [inline content/actions/managing-workflow-runs/manually-running-a-workflow.md:36]: I agree that folks landing on this page might want to know about both the workflow configuration and how to use the UI, but we do already have an example workflow for this case on another page. Both sections "Running a workflow using the REST API" and "Running a workflow on GitHub" require the workflow to be configured for the `workflow_dispatch`, which is why we've included the paragraph linking to the events at the top of the article
https://github.com/github/docs/pull/358/files#diff-0d421d891ef5301f3a44801c5f65c183d09e5ffb890dfc8c95ea68f29311a402R13

But I realize that the link we include up there doesn't directly link to the `workflow_dispatch` event. 😬 It should.

To address the visibility of that link, maybe we can create a new section header for the information linking to the `workflow_dispatch` event description. So you could replace the paragraph at the top with:

```
### Configuring a workflow to run manually

To run a workflow manually, the workflow must be configured to run on the `workflow_dispatch` event. For more information about configuring the `workflow_dispatch` event, see "[Events that trigger workflows](/actions/reference/events-that-trigger-workflows#workflow_dispatch)."
```

We could then also include your example here as an alternate example on this page: https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#example-workflow-configuration

For example:


#### Example 
```yaml
on: workflow_dispatch
```

#### Example configuration with inputs

This example defines the name and home inputs and prints them using the github.event.inputs.name and github.event.inputs.home contexts. If a name isn't provided, the default value 'Mona the Octocat' is printed.

```yaml
name: Manually triggered workflow
on:
  workflow_dispatch:
    inputs:
      name:
        description: 'Person to greet'
        required: true
        default: 'Mona the Octocat'
      home:
        description: 'location'
        required: false

jobs:
  say_hello:
    runs-on: ubuntu-latest
    steps:
    - run: |
        echo "Hello ${{ github.event.inputs.name }}!"
        echo "- in ${{ github.event.inputs.home }}!"
```

**Review by rachmari — COMMENTED** [review]
@bnb thank you for pointing out this missing information in the docs! 🙌 I think the addition you made definitely makes sense, but the article in this section are all geared toward how to accomplish something after the workflow is created. We may change that in the future, and your PR definitely prompted me to rethink that model. 

I had a suggestion that wouldn't change the model we currently have but would require some additional changes. Let me know what you think! 💚 💭

File: content/actions/managing-workflow-runs/manually-running-a-workflow.md (hunk 1 above)
Comment (bnb) [reply content/actions/managing-workflow-runs/manually-running-a-workflow.md:36]: This makes sense. Would you like me to do that work in this PR, create a new one, or would it be something y'all would want to do yourselves?

Comment (rachmari) [reply content/actions/managing-workflow-runs/manually-running-a-workflow.md:36]: @bnb feel free to update this PR if you'd like to. Our team probably won't be able to get to this for a bit, so any contributions you make here would be super helpful!

**Comment by bnb** [comment]
force pushed the changes @rachmari suggested (to the best of my understanding of them) 👍🏻

File: content/actions/reference/events-that-trigger-workflows.md [hunk 2]
```diff
@@ -98,6 +98,14 @@ You can manually trigger a workflow run using the {% data variables.product.prod
 
  To trigger the custom `workflow_dispatch` webhook event using the REST API, you must send a `POST` request to a {% data variables.product.prodname_dotcom %} API endpoint and provide the `ref` and any required `inputs`. For more information, see the "[Create a workflow dispatch event](/rest/reference/actions/#create-a-workflow-dispatch-event)" REST API endpoint.
 
+##### Example
+
+To use the `workflow_dispatch` event, you need to include it as a trigger in your GitHub Actions YAML. The example below only run a GitHub Action when it's manually triggered:
```
Comment (rachmari) [inline content/actions/reference/events-that-trigger-workflows.md:103]:
```suggestion
To use the `workflow_dispatch` event, you need to include it as a trigger in your GitHub Actions workflow file. The example below only runs the workflow when it's manually triggered:
```

**Review by rachmari — COMMENTED** [review]
@bnb this update looks great! I just had a few copy edit suggestions, but once those are fixed up I'll get this merged. 🚢 :shipit: 🚀

**Review by aj1988aj — APPROVED** [review]

**Review by rachmari — APPROVED** [review]
@bnb thanks! That was the last thing I needed. I'll get this shipped right now. 🚢 🚢 🚢

**Review by GokhanAydinnn — COMMENTED** [review]
11 nn

**Pull request merged**